config/environments/development.rb

```diff
@@ -17,44 +15,41 @@
   # Enable server timing.
   config.server_timing = true
 
-  # Enable/disable caching. By default caching is disabled.
-  # Run rails dev:cache to toggle caching.
+  # Enable/disable Action Controller caching. By default Action Controller caching is disabled.
+  # Run rails dev:cache to toggle Action Controller caching.
   if Rails.root.join("tmp/caching-dev.txt").exist?
-    config.cache_store = :memory_store
-    config.public_file_server.headers = { "Cache-Control" => "public, max-age=#{2.days.to_i}" }
+    config.public_file_server.headers = { "cache-control" => "public, max-age=#{2.days.to_i}" }
   else
     config.action_controller.perform_caching = false
-
-    config.cache_store = :null_store
   end
 
+  # Change to :null_store to avoid any caching.
+  config.cache_store = :memory_store
```

Comment:
Is this change intentional? I'm not sure how big a difference it makes but I'm wondering if we set it to `:null_store` for a reason

Reply:
yeh https://railsdiff.org/7.2.2/8.0.0#diff-4cc21c4ebed19383e79b7e232d2be4b0e6b154f2